plugins/filebrowser/pluma-file-browser-plugin.c

```diff
@@ -79,43 +80,43 @@ static void on_uri_activated_cb          (PlumaFileBrowserWidget * widget,
 static void on_error_cb                  (PlumaFileBrowserWidget * widget,
                                           guint code,
                                           gchar const *message,
-                                          PlumaFileBrowserPluginPrivate * data);
+                                          PlumaFileBrowserPluginPrivate * priv);
```

Comment:
you should also comment the refactor of the variable name from `data` to `priv` in the commit body, or you can also make this change in another commit.